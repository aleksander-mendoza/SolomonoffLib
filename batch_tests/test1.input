g = 'tert' /vis g stdout