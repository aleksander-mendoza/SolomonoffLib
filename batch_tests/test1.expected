digraph G{
3 [label="3:0",peripheries=2];
0 -> 1 [label="[d-e]:0 ''"];
0 [label="0"];
1 -> 2 [label="[q-r]:0 ''"];
1 [label="1"];
2 -> 3 [label="[s-t]:0 ''"];
2 [label="2"];
4 -> 0 [label="[s-t]:0 ''"];
4 [style=invis];
}
